perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/setup_container.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# Install go lang
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.4.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
+
+# Clone the repo and build gcsfuse
+# TODO change the branch once log_rotation is merged
+git clone "https://github.com/GoogleCloudPlatform/gcsfuse.git"
+cd gcsfuse
+git checkout log_rotation
+go build .
+cd -
+
+# Mount the bucket
+echo "Mounting the bucket"
+gcsfuse/gcsfuse --implicit-dirs --experimental-storage-client-library --max-conns-per-host 100 --disable-http2 --log-format "text" --log-file /home/logs/log.txt --stackdriver-export-interval 60s ml-models-data-gcsfuse myBucket > /home/output/gcsfuse.out 2> /home/output/gcsfuse.err &
```

Comment:
Discussed with @raj-prince , please mount with nohup like https://github.com/GoogleCloudPlatform/gcsfuse/blob/c930644995ea24683fda21219e0e001bc16bb90a/perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_container.sh#L20. 
Add a comment saying that Running in background is required so that docker doesn't keep running even after failure in python model.